topo/node/srl/srl.go

```diff
@@ -25,7 +26,9 @@ import (
 
 const (
 	scrapliPlatformName = "nokia_srl"
-	configResetCmd      = "load factory auto-commit"
+	// srl-controller v0.6.0+ creates a named checkpoint "initial" on node startup
+	// configuration reset is therefore done by reverting to this checkpoint
+	configResetCmd = "/tools system configuration checkpoint initial revert"
```

Comment:
resetting is now done by reverting to a checkpoint named `initial`.
This checkpoint is created by srl-controller:v0.6.0+ and will capture the config state of a node once it has booted and applied startup config (if present)